src/components/pages/roster-page/roster-page.tsx

```diff
@@ -370,152 +372,157 @@ export const RosterPage = () => {
       <Container maxWidth="md">
         <PageHeader title="Roster" />
 
-        <div className={classes.buttons}>
-          <input
-            accept="text/csv"
-            id="raised-button-file"
-            type="file"
-            style={{ display: 'none' }}
-            ref={fileInputRef}
-            onChange={handleFileInputChange}
-          />
-          <label htmlFor="raised-button-file">
-            <Button
+        <ErrorBoundary
```

Comment:
I wonder if we should just keep ErrorBoundary near the root of the hierarchy for now (similar to how it was before). Is there any reason you can think of that it needs to be this granular at the moment?

Reply:
Sounds good. Removed from `<RosterPage>`